### PR TITLE
Created functions to control css class names

### PR DESCRIPTION
**Pull request by rimlin**
Created functions to manage css class names

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+export function addCssClassName(element: HTMLImageElement | HTMLDivElement, cssClassName: string) {
+    if (!element.className.includes(cssClassName)) {
+        element.className += `  ${cssClassName}`
+    }
+}
+
+export function hasCssClassName(element: HTMLImageElement | HTMLDivElement, cssClassName: string) {
+    return element.className && element.className.includes(cssClassName);
+}
```
Comment (tjoskar) [inline src/utils.ts]: Even though these functions are quite simple, it should be nice to add some unit tests.

Comment (tjoskar) [reply src/utils.ts]: We do not need to test `replace` and  `includes` but the code has logic that checks if `className` is defined and that line 11 only should run if `cssClassName` don't exist.

**Comment by tjoskar** [comment]
Thanks @rimlin, can you add a few test case for the code?

**Comment by rimlin** [comment]
@tjoskar yes, will add tests at this week.

**Comment by rimlin** [comment]
Added tests.

**Comment by tjoskar** [comment]
Thanks @rimlin, I added a few missing semicolon: https://github.com/tjoskar/ng-lazyload-image/commit/d38c76c315cb06bf54e608f98e93bdfe31cacbce

**Comment by tjoskar** [comment]
Ref: #270

**Pull request merged**